tests/func/test_install.py

```diff
@@ -50,7 +50,7 @@ def test_post_checkout(self, tmp_dir, scm, dvc):
         os.unlink("file")
         scm.install()
 
-        scm.checkout("new_branch", create_new=True)
+        scm.gitpython.git.checkout("-b", "new_branch")
```

Comment:
libgit2 does not run git hooks on any command, and internally we don't need to run this hook.

For this test we want to make sure that the installed DVC post-checkout hook is run when a user does command-line `git checkout`, not that it is run on DVC `scm.checkout`.